uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -79,19 +85,32 @@ public BrokerHost(@NonNull final String brokerHostApkName,
         brokerFlightsFragment = new BrokerFlightsFragment();
         brokerApiFragment = new BrokerApiFragment();
         singleWpjApiFragment = new SingleWpjApiFragment();
+        multipleWpjApiFragment = new MultipleWpjApiFragment();
     }
 
     @Override
     public void performDeviceRegistration(@NonNull final String username,
```

Comment:
there probably should be 3 sets of methods.

1. Legacy WPJ operations
2. MWPJ Operations
3. override methods. This will either invoke 1 or 2 based on the flag.

You can put 1 and 2 into a separate class for readability.

If you're writing test specifically for MWPJ, invoke 2
If you're writing test that specifically requires legacy WPJ, invoke 1
Otherwise, invoke the override methods. (the rest of the test suites doesn't care if MWPJ or legacy WPJ is used).

Reply:
Legacy operations are contained in the Single WPJ API fragment, 
and MWPJ operations in the Multiple WPJ API fragment,
by default the broker interface uses the legacy API.

Reply:
Please add that as a comment somewhere.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2034/commits/76ecbc02d47877ec42b45ae63513912e32b63e74